src/core/server/legacy_compat/index.ts

```diff
@@ -21,21 +21,14 @@
 export { LegacyPlatformProxifier } from './legacy_platform_proxifier';
 /** @internal */
 export { LegacyConfigToRawConfigAdapter, LegacyConfig } from './legacy_platform_config';
-/** @internal */
-export { LegacyKbnServer } from './legacy_kbn_server';
 
-import {
-  LegacyConfig,
-  LegacyConfigToRawConfigAdapter,
-  LegacyKbnServer,
-  LegacyPlatformProxifier,
-} from '.';
+import { LegacyConfig, LegacyConfigToRawConfigAdapter, LegacyPlatformProxifier } from '.';
 import { BehaviorSubject, k$, map } from '../../lib/kbn_observable';
 import { Env } from '../config';
 import { Root } from '../root';
 import { BasePathProxyRoot } from '../root/base_path_proxy_root';
 
-function initEnvironment(rawKbnServer: any) {
+function initEnvironment(rawKbnServer: any, isDevClusterMaster = false) {
```

Comment:
__note:__ in the upcoming PRs, it will be detected like that: https://github.com/elastic/kibana/pull/19994/files#diff-806634e0e6f42d8ea73cd3565c9089cbR34